frontend/src/libs/components/pagination/pagination.tsx

```diff
@@ -43,7 +43,7 @@ const Pagination: React.FC<Properties> = ({
     pageIndex > 0;
 
   const isHiddenLastPage =
-    pageIndex + middleValue < convertToIndex(pageCount) &&
+    pageIndex + middleValue <= convertToIndex(pageCount) &&
```

Comment:
can you please explain why this condition was added?

Reply:
I've found a bug when dots don’t show and the reason was this condition